kubetest/federation.go

```diff
@@ -81,15 +82,41 @@ func (m *multiClusterDeployment) Enabled() bool {
 	return len(m.clusters) > 0
 }
 
+func useFederationRepo() bool {
```

Comment:
nit: add a TODO say remove all those logic in main repo once we drop testing support for federation in release branches. (probably around 1.11 release frame)